src/ic-cdk/src/api/call.rs

```diff
@@ -230,79 +278,36 @@ pub fn notify_raw(
 pub fn call_raw(
     id: Principal,
     method: &str,
-    args_raw: &[u8],
+    args_raw: Vec<u8>,
```

Comment:
`arg_raw` now takes `Vec` instead of slice. Seems to be an unnecessary interface change.
I'm aware that if we keep the API, we will have to clone the slice internally which may incur some overhead.
But I still prefers to avoid API change as much as possible.
@adamspofford-dfinity  WDYT?

Reply:
I specifically requested this. Avoiding API change is not worth the potential overhead of a possibly multi-megabyte unnecessary copy.

Reply:
However, I think this API is capable of working with borrowed types. Let me do some testing.